tests/integration/targets/nxos_static_routes/tasks/cli.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: collect common test cases
+  find:
+    paths: "{{ role_path }}/tests/common"
+    patterns: "{{ testcase }}.yaml"
```

Comment:
Check for .yaml and .yml differences in test file names and make them consistent.